README.md

```diff
@@ -52,9 +67,9 @@ import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
 ```
 
 
-## Add Entity tab for Maturity
+## Add Tab for Maturity to Services
 
-Add to the `serviceEntityPage` in `packages/app/src/components/catalog/EntityPage.tsx`
+In `packages/app/src/components/catalog/EntityPage.tsx` import the plugin and add it to `serviceEntityPage`
```

Comment:
I found I often misread this line. The established pattern from the previous instructions was filename first, so I updated this to match.